app/common/dispatcher/dispatchAction.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function (name, action) {
```

Comment:
in the short term it's confusing, but we need to split out the actual dispatcher functionality from the code that uses it so we can include it in different contexts like worker threads. I'm open to name changes if you have thoughts...